app/code/core/Mage/Core/etc/config.xml

```diff
@@ -122,7 +122,7 @@
             <types>
                 <config translate="label,description" module="core">
                     <label>Configuration</label>
-                    <description>System(config.xml, local.xml) and modules configuration files(config.xml).</description>
+                    <description>System(config.xml, local.xml) and modules configuration files(config.xml, system.xml).</description>
```

Comment:
I suggest to add the missing spaces:
`System (config.xml, local.xml) and modules configuration files (config.xml, system.xml).`

Reply:
:'(

Reply:
Will you merge my stuff? :)

Reply:
I can't, but someone perhaps.
I will test it again... with the missing spaces? :)

Reply:
Added whitespaces. Welcome to merge

Reply:
I just realized that the description is translated, so changing it will require changing the translation as well, and that'll break language packs.

Reply:
It seems that the original text is missing into _Mage_Core.csv_. Can you add the new one into the CSV?

Reply:
Text added.